cmd/snap-update-ns/sorting_test.go

```diff
@@ -52,21 +52,60 @@ func (s *sortSuite) TestTrailingSlashesComparison(c *C) {
 func (s *sortSuite) TestParallelInstancesAndSimple(c *C) {
 	// Naively sorted entries.
 	entries := []osutil.MountEntry{
-		{Dir: "/a/b"},
 		{Dir: "/a/b-1"},
+		{Dir: "/a/b", Options: []string{osutil.XSnapdOriginLayout()}},
+		// this would get sorted before /var/bar if looked at name only
```

Comment:
```suggestion
		// this would get sorted before /var/snap if looked at name only
```

Reply:
actually the comment is wrong, so I'll just drop it